libs/language-server/src/lib/ast/expressions/evaluation.ts

```diff
@@ -56,18 +62,29 @@ export type InternalValueRepresentation =
 
 export class EvaluationContext {
   constructor(
-    public runtimeParameterValues: Map<
+    private readonly runtimeParameterValues: Map<
+      string,
+      InternalValueRepresentation
+    > = new Map(),
+    private readonly variableValues: Map<
       string,
       InternalValueRepresentation
     > = new Map(),
-    public variableValues: Map<string, InternalValueRepresentation> = new Map(),
+    private valueKeywordValue:
```

Comment:
I assume that we have this distinguished handling due to the `length` property, right? Otherwise we could just add it to the `varuableValues` map with the key `value`.

Reply:
Yes indeed :+1: